scripts/obtain/dotnet-install.ps1

```diff
@@ -10,18 +10,14 @@
     Installs dotnet cli. If dotnet installation already exists in the given directory
     it will update it only if the requested version differs from the one already installed.
 .PARAMETER Channel
-    Default: preview
-    Channel is the way of reasoning about stability and quality of dotnet. This parameter takes one of the values:
-    - future - Possibly unstable, frequently changing, may contain new finished and unfinished features
-    - preview - Pre-release stable with known issues and feature gaps
-    - production - Most stable releases
+    Default: master
```

Comment:
"master"
"release/2.0.0"
The above referenced channels are the only ones that are viable* at this time: 5/23/2017
*viable defined as existing at the standard blob storage location for both SDK & the Runtime
See: 
https://dotnetcli.blob.core.windows.net/dotnet/Runtime/release/2.0.0/
https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master
https://dotnetcli.blob.core.windows.net/dotnet/Sdk/release/2.0.0/
https://dotnetcli.blob.core.windows.net/dotnet/Sdk/master/